.travis.yml

```diff
@@ -32,7 +33,6 @@ before_install:
     - source venv/bin/activate
     - python --version # just to check
     - retry pip install nose # always
-    - sudo apt-get install libblas-dev liblapack-dev libatlas3gf-base
```

Comment:
OK - maybe these get installed with the numpy installations.

But - I think you will need some sort of pip cacheing: https://github.com/scikit-image/scikit-image/pull/1610